src/store/savedFilter/slice.ts

```diff
@@ -20,7 +20,17 @@ export const SavedFilterState: initialState = {
 };
 
 const sortByUpdateDate = (sets: TUserSavedFilter[]) =>
-  sets.sort((a, b) => (new Date(a.updated_date) < new Date(b.updated_date) ? 0 : -1));
+  sets.sort((a, b) => {
```

Comment:
1. could you extract this sorting body for reuse
2. (to verify) I would be tempted to add a `.slice` before the sort: `sets.slice().sort(...)` to be sure we don t mutate `sets`. I don t believe it s an issue, but just to be sure...